ex_webrtc/lib/ex_webrtc_endpoint.ex

```diff
@@ -436,8 +441,8 @@ defmodule Membrane.RTC.Engine.Endpoint.ExWebRTC do
     end)
 
     tracks_ready =
-      Enum.map(tracks, fn track ->
-        {:notify_parent, {:track_ready, track.id, :high, track.encoding}}
+      Enum.flat_map(tracks, fn track ->
+        Enum.map(track.variants, &{:notify_parent, {:track_ready, track.id, &1, track.encoding}})
```

Comment:
```suggestion
        Enum.map(track.variants, &{:notify_parent, {:track_ready, track.id, &1, track.variant}})
```

Reply:
This is alright, the `&1` is the variant, the encoding is actually the encoding:
```
@type track_ready_action_t() ::
          {:notify_parent, {:track_ready, Track.id(), Track.variant(), Track.encoding()}}
```